rest_framework_simplejwt/views.py

```diff
@@ -84,3 +176,35 @@ class TokenVerifyView(TokenViewBase):
 
 
 token_verify = TokenVerifyView.as_view()
+
+
+class TokenCookieDeleteView(APIView):
+    """
+    Deletes httpOnly auth cookies.
+    Used as logout view while using AUTH_COOKIE
+    """
+    authentication_classes = ()
+    permission_classes = ()
+
+    def post(self, request):
+        response = Response({})
+
+        if api_settings.AUTH_COOKIE:
+            self.delete_auth_cookies(response)
+
+        return response
+
+    def delete_auth_cookies(self, response):
+        response.delete_cookie(
+            api_settings.AUTH_COOKIE,
+            domain=api_settings.AUTH_COOKIE_DOMAIN,
+            path=api_settings.AUTH_COOKIE_PATH
+        )
+        response.delete_cookie(
+            '{}_refresh'.format(api_settings.AUTH_COOKIE),
+            domain=None,
+            path=reverse('token_refresh'),
+        )
+
```

Comment:
I think you should emit

django.contrib.auth.signals.user_logged_in here

(see https://github.com/davesque/django-rest-framework-simplejwt/issues/190)

Reply:
It is not a subject of this PR and #196 has not yet been merged. Let me know when it will be accepted.

Reply:
I will, but this one will probably go first. Thanks